src/cron/clear_transactions.rs

```diff
@@ -0,0 +1,14 @@
+use crate::{database::{queries::{QueryContext, delete_txs}}};
+
+use super::CronJobError;
+use crate::cron::ValidatorCronError;
+
+pub async fn clear_old_transactions<Context>(ctx: &Context) -> Result<(), CronJobError>
+where
+  Context: QueryContext 
+{
+  let epoch = ctx.current_epoch();
+  delete_txs(ctx, epoch, 40).await
```

Comment:
40 is probably too small here, I think we should use the same value that the tx is expected to be included in a Arweave tx (the bundler) and maybe add few blocks on top of it.

Reply:
@joshbenaron any comments on this one?